feat(interlink): private beta doc (#3942)
scaleway/docs-content: network/interlink/reference-content/overview.mdx

<Lightbox src="scaleway-interlink-diag-tech.webp" alt="Diagram of Scaleway’s network architecture for connecting customer infrastructure via a partner using a hosted InterLink. On the left, Scaleway’s PAR region includes a VPC with an InterLink and vRouter connected to the PAR backbone. This links to a Point of Presence (PoP) in Paris (TH2), where both Scaleway and partner routers are located. The partner’s infrastructure connects the Paris PoP to a customer or partner PoP, such as in Marseille. The customer PoP has a partner router and a customer router connected through an MMR (Meet-Me Room). The setup supports either on-premises customer infrastructure via last-mile connection or hosted customer infrastructure in the partner’s data center" />

### Dedicated InterLink

A dedicated InterLink is provisioned, where the customer gets sole use of a single InterLink port on Scaleway's router, with the entire bandwidth of that port. This type of InterLink is not yet available at Scaleway - [contact a sales representative](https://www.scaleway.com/en/contact-sales/) to register your interest.

## Provisioning an InterLink

> Provisioning an InterLink is a multi-step process. As only hosted InterLinks (via partners) are currently available, specific action is required from both from Scaleway, the customer, and from the selected partner in order to successfully complete the provisioning process. See our dedicated page on [provisioning an InterLink](/network/interlink/reference-content/provisioning/) for full details.

Reviewer: ```suggestion
Provisioning an InterLink is a multi-step process. As only hosted InterLinks (via partners) are currently available, specific action is required from Scaleway, the customer, and the selected partner in order to successfully complete the provisioning process. See our dedicated page on [provisioning an InterLink](/network/interlink/reference-content/provisioning/) for full details.
```

## Attaching a VPC

> When you initially create the InterLink, it is not associated with a specific VPC, only with given a Scaleway region. Attaching it to a VPC within the specified region is essential to complete the connection on the Scaleway side. You can only attach a VPC once the provisioning process is complete, and the InterLink has `Active` status. See the [configuration](/network/interlink/reference-content/configuring/) page for more information.

Reviewer: ```suggestion
When you initially create the InterLink, it is not associated with a specific VPC, only with a given Scaleway region. Attaching it to a VPC within the specified region is essential to complete the connection on the Scaleway side. You can only attach a VPC once the provisioning process is complete, and the InterLink has `Active` status. See the [configuration](/network/interlink/reference-content/configuring/) page for more information.
```

### Allowed routes list

By default, **all route announcements from both sides are blocked** by InterLink. This is part of a safety-net mechanism to allow customers to efficiently and safely control traffic flow across the link. You must create and attach an [allowed routes list](/network/interlink/concepts/#allowed-routes-list) (aka routing policy), to specify the IP prefix announcements to whitelist from each side. This is step one of two in allowing traffic to flow across the InterLink. See the [configuration](/network/interlink/reference-content/configuring/) page for more information on allowed routes lists. 

### Route propagation

Even after attaching an allowed route list, traffic cannot flow across your InterLink until you **activate route propagation**. This is the second part of the safety-net mechanism enabling you to fine-tune the traffic that flows across your InterLink. Route propagation can be activated or deactivated whenever you want. See the [configuration](/network/interlink/reference-content/configuring/) page for more information on route propagation. 

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

---
meta:

>   title: InterLink - technical overview

Reviewer 2: ```suggestion
  title: InterLink - Technical overview
```

  description: Dive deeper into understanding Scaleway's InterLink product, with technical diagrams, explanations and more.
  noindex: true
content:

>   h1: InterLink - technical overview

Reviewer 2: ```suggestion
  h1: InterLink - Technical overview
```

  validation: 2024-10-24
  posted: 2024-10-24
---

<Message type="important">
InterLink is currently in [Private Beta](https://www.scaleway.com/en/betas/), and only available to selected testers.
</Message>

InterLink is Scaleway's product to let you create a secure, private connection between your external infrastructure, and your Scaleway VPC. This allows you to direct your traffic safely from your Scaleway infrastructure to your on-premises infrastructure, away from the public internet. Compared to a typical internet-based VPN, an InterLink is more reliable, more secure and offers lower latency.

At its simplest level, InterLink can be understood as a configurable connection between your own infrastructure, and your Scaleway VPC. You can configure details such as available bandwidth, and the routes allowed between the two peers:

<Lightbox src="scaleway-interlink-diag-simple.webp" alt="A diagram shows how a Scaleway VPC, to the left, can be attached to an InterLink, which then connects to external customer infrastructure." />

## Types of InterLink

Scaleway plans to offer three types of InterLink, though currently only the **hosted** type is available.

### Hosted InterLinks

> A hosted InterLink is provisioned via a [partner](/network/interlink/concepts/#partner). Partners are telcos or network providers that have been pre-selected by Scaleway. They provide a shareable port on their router, with a certain amount of bandwidth, at a [PoP](/network/interlink/concepts/#location) (location) where both Scaleway and the partner's networks are present. The available bandwidth is shared between multiple customers: each gets a specified amount when creating their InterLink.

Reviewer 2: ```suggestion
A hosted InterLink is provisioned via a [partner](/network/interlink/concepts/#partner). Partners are telecom operators or network providers that have been pre-selected by Scaleway. They provide a shareable port on their router, with a certain amount of bandwidth, at a [PoP](/network/interlink/concepts/#location) (location) where both Scaleway and the partner's networks are present. The available bandwidth is shared between multiple customers: each gets a specified amount when creating their InterLink.
```

### Allowed routes list

By default, **all route announcements from both sides are blocked** by InterLink. This is part of a safety-net mechanism to allow customers to efficiently and safely control traffic flow across the link. You must create and attach an [allowed routes list](/network/interlink/concepts/#allowed-routes-list) (aka routing policy), to specify the IP prefix announcements to whitelist from each side. This is step one of two in allowing traffic to flow across the InterLink. See the [configuration](/network/interlink/reference-content/configuring/) page for more information on allowed routes lists. 

### Route propagation

Even after attaching an allowed route list, traffic cannot flow across your InterLink until you **activate route propagation**. This is the second part of the safety-net mechanism enabling you to fine-tune the traffic that flows across your InterLink. Route propagation can be activated or deactivated whenever you want. See the [configuration](/network/interlink/reference-content/configuring/) page for more information on route propagation. 

> 

Reviewer 2: Remove additional empty lines at the end of file
